### ISSUE TITLE
[Plugin] Terraform Runner

### ISSUE THREAD

**Issue by juliosueiras** [backstage/backstage#690]
## Summary

A Plugin that would allow running arbitrary terraform code/modules, possible use case is for service catalog or direct run on a service page   

## Project website (if applicable)

https://terraform.io

**Comment by kenrose** [comment]
Related: [Atlantis](https://www.runatlantis.io/) can do this as part of CI.  Maybe worthwhile thinking of this as an Atlantis plugin.

**Comment by OrkoHunter** [comment]
Re-opening since plugin suggestion issues are now exempt from [stalebot](https://github.com/backstage/backstage/blob/18f7345a6c29dcfccef7df081b1708f2bf8f62ff/.github/stale.yml#L9).

**Comment by asarenkansah** [comment]
Is anyone still working on this?

**Comment by benjdlambert** [comment]
I don't think that this has been picked up by anyone yet. Would be good if we could get some of the hashicorp devs or someone with a lot of experience with terrarform to implement something that can visualize state etc in a plugin :pray:

**Comment by gumartinm** [comment]
I guess, what needs to be implemented is a plugin for Atlantis as it was mentioned in a comment 2 years ago.

**Comment by rawIce** [comment]
if this or a variation of Atlantis was implemented, our company would love to consider backstage as our internal developer platform
